File: examples/randomPlayer.cpp
#include<iostream>
#include<algorithm>
#include<vector>

int main(){
    // read n and j
    int n, j;
    std::cin >> n >> j;

    // Make a vector of all cards 1,2,3,4,....,13,14,15
    std::vector<int> cards = {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15};

    // shuffling the cards
    std::random_shuffle(cards.begin(), cards.end());

    // laying out these cards in this order
    for (int card : cards){
        // reading winnable card in the middle (and ignoring it)
        int w;
        std::cin >> w;

        // play this card
        std::cout << card << std::endl;

        // reading the cards the other players played (and ignoring it)
        std::vector<int> playedCards(n);
        for (int i = 0; i < n; i++){
            std::cin >> playedCards[i];
        }
    }
}

File: examples/randomPlayer.py
# import random to get randomness
from random import shuffle

# read n
n = int(input())

# Make a list of all cards 1,2,3,4,....,13,14,15
cards = list(range(1,16))

# shuffling the cards
shuffle(cards)

# laying out these cards in this order
for card in cards:
    # reading winnable card in the middle (and ignoring it)
    w = int(input())

    # play this card
    print(card)

    # reading the cards the other players played (and ignoring it)
    playedCards = list(map(int, input().split()))
